x-pack/plugins/actions/server/builtin_action_types/swimlane/service.ts

```diff
@@ -0,0 +1,148 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { Logger } from '@kbn/logging';
+import axios from 'axios';
+import https from 'https';
+import { ProxySettings } from '../../types';
+import {
+  SwimlanePublicConfigurationType,
+  ExternalService,
+  ExternalServiceCredentials,
+  CreateRecordParams,
+  CreateRecordResponse,
+  SwimlaneSecretConfigurationType,
+  MappingConfigType,
+  SwimlaneRecordPayload,
+  GetApplicationResponse,
+} from './types';
+import * as i18n from './translations';
+import { getErrorMessage, request } from '../lib/axios_utils';
+
+export const createExternalService = (
+  { config, secrets }: ExternalServiceCredentials,
+  logger: Logger,
+  proxySettings?: ProxySettings
+): ExternalService => {
+  const { apiUrl: url, appId, mappings } = config as SwimlanePublicConfigurationType;
+  const { apiToken } = secrets as SwimlaneSecretConfigurationType;
+
+  const axiosInstance = axios.create({
+    httpsAgent: new https.Agent({
+      rejectUnauthorized: false,
```

Comment:
we cannot have this `rejectUnauthorized` in production

Reply:
What's the best way to keep that local only?

Reply:
I am not sure to be honest. I usually do the same but I did not commit the file.